dsn/dsn.go

```diff
@@ -0,0 +1,64 @@
+package dsn
+
+import (
+	"fmt"
+)
+
+// Usage of this package
+//
+// db, err := ydb.New(
+//   ctx,
+//   ydb.WithConnectionString(
+//     New("endpoint", "database").WithInsecure().String(),
+// ),
+//)
+
+// Maker helps to make connection string from separated endpoint and database
+type Maker interface {
+	fmt.Stringer
+
+	// WithSecure makes new Maker from current Maker with passed secure flag
+	WithSecure(secure bool) Maker
+
+	// WithInsecure makes new insecured Maker from current Maker
+	WithInsecure() Maker
```

Comment:
why do you need to add extra call equivalent to `WithSecure(false)`?

Reply:
ok, I will remove it